File: src/tennis_soundboard/SoundHandler.java
package tennis_soundboard;

import java.applet.Applet;
import java.applet.AudioClip;
import java.io.File;
import java.net.MalformedURLException;
import java.net.URL;


public class SoundHandler {
    
    public void playSound(String sound) {
        try {
            File file = new File(("src/tennis_soundboard/audio/" + sound + ".wav"));
            URL url = null;
            if (file.canRead()) {
                url = file.toURI().toURL();
            } else {
                System.out.println("Could not read file.");
                System.out.println(file.getAbsoluteFile());
            }
            System.out.println(url);
            AudioClip clip = Applet.newAudioClip(url);
            clip.play();
            System.out.println("playing file: " + sound + ".wav");
        } catch (MalformedURLException e) {
            System.out.println("ERROR");
        }
    }
}

File: src/tennis_soundboard/SoundHandler.kt
package tennis_soundboard


import java.io.File
import javax.sound.sampled.AudioSystem

class SoundHandler {
    fun playSound(soundFile: String) {
        val f = File("./res/audio/$soundFile.wav")
        val audioIn = AudioSystem.getAudioInputStream(f.toURI().toURL())
        val clip = AudioSystem.getClip()
        clip.open(audioIn)
        clip.start()
    }
}